src/pkg/services/m365/api/events.go

```diff
@@ -192,6 +192,25 @@ func (c Events) PatchCalendar(
 	return nil
 }
 
+func (c Events) GetCalendarEvents(
```

Comment:
if this is only used for testing it should be defined in the test file as perhaps a wrapper around the `Events` struct (if access to otherwise private data is required). We should generally avoid adding extra functions to a struct/interface if they're only used in testing

Reply:
fwiw, this should probably get swapped out for the item enumerator being introduced for advanced restore options, too.